perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -40,15 +41,49 @@
 
 NUM_KEYS_PER_CORE = 2000000
 
+# TODO: Support read/counter_read/mixed command.
+# cassandra-stress command
+WRITE_COMMAND = 'write'
+COUNTER_WRITE_COMMAND = 'counter_write'
+USER_COMMAND = 'user'
+
+# cassandra-stress command [options]
+flags.DEFINE_enum('cassandra_stress_command', WRITE_COMMAND,
+                  [WRITE_COMMAND,
+                   COUNTER_WRITE_COMMAND,
+                   USER_COMMAND],
+                  'cassandra-stress command to use.')
+
+# Options for cassandra-stress
 flags.DEFINE_integer('num_keys', 0,
-                     'Number of keys used in cassandra-stress tool. '
-                     'If unset, this benchmark will use %s * num_cpus '
-                     'on data nodes as the value.' % NUM_KEYS_PER_CORE)
+                     'Number of keys used in cassandra-stress tool across '
+                     'all loader vms. If unset, this benchmark will use '
+                     '%s * num_cpus on data nodes as the value.'
+                     % NUM_KEYS_PER_CORE)
 
 flags.DEFINE_integer('num_cassandra_stress_threads', 50,
                      'Number of threads used in cassandra-stress tool '
                      'on each loader node.')
 
+flags.DEFINE_integer('cassandra_stress_replication_factor', 3,
+                     'Number of replicas.')
+
+flags.DEFINE_enum('cassandra_stress_consistency_level', 'QUORUM',
+                  ['ONE', 'QUORUM', 'LOCAL_ONE', 'LOCAL_QUORUM',
+                   'EACH_QUORUM', 'ALL', 'ANY'],
+                  'Set the consistency level to use during cassandra-stress.')
+
+flags.DEFINE_integer('cassandra_stress_retries', 1000,
+                     'Number of retries when error encountered during stress.')
+
+# Options to use with cassandra-stress user mode.
+# http://www.datastax.com/dev/blog/improved-cassandra-2-1-stress-tool-benchmark-any-schema
+flags.DEFINE_string('cassandra_stress_profile', '',
```

Comment:
I see you mentioned this in a comment above, but might be nice to explicitly mention that these flags only matter if --cassandra_stress_command is "user" in the flag help text so users using --help wont get confused